stdlib/src/md5.rs

```diff
@@ -0,0 +1,13 @@
+pub(crate) use _md5::make_module;
+
+#[pymodule]
+mod _md5 {
+    use crate::hashlib::_hashlib::{HashArgs, HashWrapper, PyHasher};
+    use crate::vm::{PyObjectRef, PyPayload, PyResult, VirtualMachine};
+    use md5::Md5;
+
+    #[pyfunction(name = "md5")]
+    fn md5(args: HashArgs, vm: &VirtualMachine) -> PyResult<PyObjectRef> {
```

Comment:
```suggestion
    fn md5(args: HashArgs, vm: &VirtualMachine) -> PyResult {
```
PyResult = PyResult<PyObjectRef>